src/server.js

```diff
@@ -124,8 +115,7 @@ server.post('/registration', checkNotAuthenticated,
             } catch (error) {
                 console.log(error)
                 req.flash("error", "User is already registered. Please login.");
-                res.redirect("/registration")//todo, results in a 302 status redirection code
-
+                res.redirect("/registration")
```

Comment:
want to keep prev TODO comment

Reply:
not sure whos comment this is but looks okay